docs/markdown/state-management/ngrx/state.md

```diff
@@ -0,0 +1,27 @@
+<!-- .slide: class="sfeir-basic-slide with-code" -->
+# NGRX: State
+<br>
+<ul>
+    <li>mapping de clé-valeur</li><br>
+    <li>récupérer grâce à nos sélectors</li><br>
+    <li>retourne toujours un observable</li>
+</ul>
+```typescript
+export interface State {
+  people: Object[];
+  search: string;
+}
+
+export interface PeopleFeature {
+  people: State;
+}
+
+export const initialState = {
+  people: [],
+  search: ''
+};
+```
```

Comment:
make the text and the code block side by side too let more space to the code and increase the font size